packages/grafana-ui/src/components/VizLegend/VizLegendTable.tsx

```diff
@@ -65,6 +65,9 @@ export const VizLegendTable = <T extends unknown>({
 
   return (
     <table className={cx(styles.table, className)}>
+      <colgroup>
+        <col style={{ width: '100%' }} />
+      </colgroup>
```

Comment:
This works, but I wonder why? :D

Reply:
According to w3schools: 
```
The <colgroup> tag is useful for applying styles to entire columns, instead of repeating the styles for each cell, for each row.
```
But I think I can just explicitly set the width of the first column to 100%. Would push the changes now